src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -0,0 +1,97 @@
+package juja.sqlcmd;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class DatabaseManagerTest {
+
+    private static final String DB_NAME = "sqlcmd";
+    private static final String TEST_DB_NAME = "test";
+    private static final String DB_USER_NAME = "sqlcmd";
+    private static final String DB_USER_PASSWORD = "sqlcmd";
+    private static final String JDBC_URL = "jdbc:postgresql://127.0.0.1:5432/";
+    private static Connection connection;
+
+    private DatabaseManager databaseManager;
+
+    @BeforeClass
+    public static void setTestingEnvironment() throws SQLException {
+        connection = DriverManager.getConnection(
+                JDBC_URL, DB_USER_NAME, DB_USER_PASSWORD);
+        executeSqlQuery("DROP DATABASE IF EXISTS " + TEST_DB_NAME);
+        executeSqlQuery("CREATE DATABASE " + TEST_DB_NAME);
+        connection.close();
+        connection = DriverManager.getConnection(
+                JDBC_URL + TEST_DB_NAME, DB_USER_NAME, DB_USER_PASSWORD);
+    }
+
+    @AfterClass
+    public static void closeConnection() throws SQLException {
+        connection.close();
+    }
+
+    @Before
+    public void setUp() throws SQLException {
+        databaseManager = new DatabaseManager();
+        databaseManager.connect(TEST_DB_NAME, DB_USER_NAME, DB_USER_PASSWORD);
+        dropAllTables();
+    }
+
+    private static void executeSqlQuery(String sqlQuery) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            statement.execute(sqlQuery);
+        }
+    }
+
+    private static void dropAllTables() throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            String sqlQuery = "DROP TABLE IF EXISTS CASCADE";
+            statement.executeUpdate(sqlQuery);
```

Comment:
This method doesn't remove any table, because you don't pass in tableName to the sql query. You should either drop database, schema or table.
Because of this method, your test "getTableNamesWhenThereAreNoTablesInDatabaseShouldReturnEmptyArray" fails, when you run tests twice

Reply:
Thanks. You are right. I have already changed it. :)